### ISSUE TITLE
Citation in Recent Paper

### ISSUE THREAD

**Issue by davibarreira** [JuliaOptimalTransport/OptimalTransport.jl#77]
Hey guys, just saw this article [here](https://jmlr.org/papers/volume22/20-451/20-451.pdf).
It seems our package is getting noticed. ;)

**Comment by devmotion** [comment]
Oh nice :tada:

But to be honest, it's not completely correct that there is no GPU implementation in OptimalTransport.jl - there's no dedicated implementation of `sinkhorn` for GPUs since the standard implementation already works on the GPU.

**Comment by davibarreira** [comment]
Yeah, I just said the same thing to a friend of mine to whom I showed the paper.

**Comment by davibarreira** [comment]
Perhaps we should point out in the documentation that one can easily use `CUDA.jl` for `sinkhorn`. What do you think?

**Comment by devmotion** [comment]
Yes, I thought the same. Maybe we could mention GPU support in the README and add the more detailed example to the documentation. I also thought it might be good to add the package(s) to Zenodo, then we can include a badge with a DOI in the README as well.

**Comment by devmotion** [comment]
StochasticOptimalTransport was already added to Zenodo (apparently had done this a while ago), I added OptimalTransport and PythonOT now as well. The next time we make a release Zenodo will generate  a DOI for us.
